fuels-signers/src/lib.rs

```diff
@@ -48,26 +46,26 @@ mod tests {
         let mut secret_seed = [0u8; 32];
         rng.fill_bytes(&mut secret_seed);
 
-        let secret =
-            SecretKey::from_slice(&secret_seed).expect("Failed to generate random secret!");
+        let secret = unsafe { SecretKey::from_bytes_unchecked(secret_seed) };
```

Comment:
This should be both commented to explain why we are doing an `unsafe` block, and IMO we should figure out the proper way to make it non-unsafe. We can't really compromise memory safety (even in this trivial case) for something like signing.

Reply:
The proper way would be via [TryFrom](https://docs.rs/fuel-crypto/0.4.2/fuel_crypto/struct.SecretKey.html#impl-TryFrom%3CBytes32%3E).

However, from the lines above, looks like you are creating a new random key. Hence, in this case, the proper way would be to call [Secret::random](https://docs.rs/fuel-crypto/0.4.2/src/fuel_crypto/secret.rs.html#122)